nx.json

```diff
@@ -12,9 +12,6 @@
       "dependsOn": ["^build"],
       "outputs": ["{projectRoot}/lib"]
     },
```

Comment:
Why are we removing the dependency test dependency on build

Reply:
Because it's not needed if tests don't require a build upfront in this case or in the case you are developing